### PR TITLE
fix width and height metadata condition on svg_dimensions function

### PR DESCRIPTION
**Pull request by sksaju**
### Description of the Change

Remediation work on the PR https://github.com/10up/safe-svg/pull/154 

Closes #

### Changelog Entry

> Added - New feature
> Changed - Existing functionality
> Deprecated - Soon-to-be removed feature
> Removed - Feature
> Fixed - Bug fix
> Security - Vulnerability

### Credits

Props @username, @username2, ...

### Checklist:

- [ ] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

File: tests/unit/test-safe-svg.php [hunk 1]
```diff
@@ -262,6 +262,53 @@ public function test_one_pixel_fix() {
 							),
 						),
 					),
+					array(
```
Comment (peterwilsoncc) [inline tests/unit/test-safe-svg.php:265]: Is this what `wp_get_attachment_metadata()` is expected to return following your changes in this and the other PR? 

Using `svgNoDimensions.svg` I don't get a sizes array stored.

**Review by peterwilsoncc — COMMENTED** [review]
The change looks good but I left a question inline about the tests and differences I am seeing locally.

File: tests/unit/test-safe-svg.php (hunk 1 above)
Comment (iamdharmesh) [reply tests/unit/test-safe-svg.php:265]: @peterwilsoncc I have updated this as per the actual response from the WP for the `svgNoDimensions.svg` file.

**Review by iamdharmesh — APPROVED** [review]
LGTM.

**Pull request merged**